lib/playerlist.js

```diff
@@ -85,54 +109,9 @@ class PlayerList extends LitElement {
       }
     })
 
-    function addPlayer (player) {
-      const playerEntry = document.createElement('div') // There is likely a better way to handle this CSS-wise
-
-      const playerPingContainer = document.createElement('div')
-      const playerPingValue = document.createElement('p')
-      const playerPingLabel = document.createElement('p')
-
-      playerEntry.classList.add('playerlist-entry')
-      playerEntry.innerHTML = player.username
-      playerEntry.id = `plist-player-${player.uuid}`
-
-      if (player.uuid === bot.player.uuid) {
-        playerEntry.classList.add('plist-active-player')
-      }
-
-      playerPingContainer.classList.add('plist-ping-container')
-
-      playerPingValue.classList.add('plist-ping-value')
-      playerPingValue.innerHTML = player.ping
-
-      playerPingLabel.classList.add('plist-ping-label')
-      playerPingLabel.innerHTML = 'ms'
-
-      playerPingContainer.appendChild(playerPingValue)
-      playerPingContainer.appendChild(playerPingLabel)
-      playerEntry.appendChild(playerPingContainer)
-      playerList.appendChild(playerEntry)
-
-      return playerEntry
-    }
-
-    function removePlayer (player) {
-      playerList.querySelector(`#plist-player-${player.uuid}`)?.remove()
-    }
-
-    bot.on('playerUpdated', player => {
-      let playerEntry = playerList.querySelector(`#plist-player-${player.uuid}`)
-
-      if (!playerEntry) {
-        playerEntry = addPlayer(player)
-      }
-
-      playerEntry.querySelector('.plist-ping-value').innerHTML = player.ping
-    })
-
-    bot.on('playerJoined', addPlayer)
-
-    bot.on('playerLeft', removePlayer)
+    bot.on('playerUpdated', () => this.requestUpdate()) // LitElement seems to be batching requests, so it should be fine?
```

Comment:
`bot.on('playerUpdated', () => this.requestUpdate())` === `bot.on('playerUpdated', this.requestUpdate)`

Reply:
this will work for all of the ones below

Reply:
Won't change, it's not ===, try doing that and see what happens to the context (and observe `Uncaught TypeError: this.requestUpdateInternal is not a function` in the console)
We discussed it in #99, and it was mentioned by @extremeheat, the same reason it's good there, it's very bad here

Reply:
Oh, I see

Reply:
In new JS engines, you can declare arrow functions at class level,
```js
class Class {
  onEvent = (evt) => alert('hello')
}
```

standardjs's parser doesn't support this yet though, so you have to do either bind `this` or do what you did here (only downside here is you can't unregister the event, but not a concern here)